.github/workflows/publish-ios.yml

```diff
@@ -16,25 +16,17 @@ jobs:
         plugin: ${{ fromJson(github.event.inputs.plugins) }}
     steps:
       - run: sudo xcode-select --switch /Applications/Xcode_13.3.1.app
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
-      - uses: actions/checkout@v2
+          node-version: 16
```

Comment:
It shouldn't matter as we aren't doing `npm install` but I'd leave it as `14.x` as the `npm` version that comes with can change the behaviour of peer deps.

Reply:
I had thought 14 was EOL

Reply:
Looks like it still has 5 months of security support left: https://endoflife.date/nodejs